tasks/sep/026-fp-holocene-upgrade-fix/NestedSignFromJson.s.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+import {console2 as console} from "forge-std/console2.sol";
+import {Vm} from "forge-std/Vm.sol";
+import {stdJson} from "forge-std/StdJson.sol";
+import {Simulation} from "@base-contracts/script/universal/Simulation.sol";
+import {NestedSignFromJson as OriginalNestedSignFromJson} from "script/NestedSignFromJson.s.sol";
+import {DisputeGameUpgrade} from "script/verification/DisputeGameUpgrade.s.sol";
+import {CouncilFoundationNestedSign} from "script/verification/CouncilFoundationNestedSign.s.sol";
+import {SuperchainRegistry} from "script/verification/Verification.s.sol";
+
+contract NestedSignFromJson is OriginalNestedSignFromJson, CouncilFoundationNestedSign, DisputeGameUpgrade {
+    constructor()
+        SuperchainRegistry("sepolia", "op", "v1.8.0-rc.4")
+        DisputeGameUpgrade(
+            0x03b7eaa4e3cbce90381921a4b48008f4769871d64f93d113fcadca08ecee503b, // absolutePrestate
```

Comment:
I have verified against the MT FDG.  https://sepolia.etherscan.io/address/0x833a817eF459f4eCdB83Fc5A4Bf04d09A4e83f3F#readContract
But I don't know if there is another place to get the information from?

Reply:
It's the mt64 prestate for the `op-program/v1.4.0-rc.3` release. Which supports op-sepolia holocene.
Documented here - https://www.notion.so/oplabs/OP-Sepolia-v1-9-0-rc-3-MT-Cannon-15af153ee16280e58414dbed970d43ef.

Reply:
Thaanks!